src/rust/ide/enso-protocol/project-manager/README.md

```diff
@@ -0,0 +1,136 @@
+File Manager consists of pair server and client. They communicate by exchaning
```

Comment:
Note to self: This file was copied from file-manager. Don't forget to update it.